### PR TITLE
Updated config.py to latest protocol

### PR DESCRIPTION
**Pull request by FrederickGeek8**

### CODE REVIEW

**Comment by lukleh** [comment]
Hi, thanks for the pull request. All current dev effort goes into https://github.com/lukleh/TwistedBot/tree/spaces that I just pushed in.

**Comment by lukleh** [comment]
I have commented on each commit and now I can't see the comments :/

**Comment by FrederickGeek8** [comment]
Sorry I deleted the repo (not thinking properly today), then reforked it, then made changes to spaces.

**Comment by lukleh** [comment]
no problem, just wasn't sure you've read the comment on generating the bot name

**Comment by FrederickGeek8** [comment]
0_0 Could you repost that please?

**Comment by lukleh** [comment]
About generating bot name: rather that having something completely random, I would prefer USERNAME =  COMMANDER + "_bot"
If one nice day people will use more that one bot on the same server, it would be more clearer which bot belongs to who.
For now I call it twistedbot. Right now I am having more pressing problems to solve than naming so this is low priority.

**Comment by FrederickGeek8** [comment]
For multiple bots how about USERNAME = COMMANDER+"_bot-"+id_generator()

**Comment by lukleh** [comment]
With current design each COMMANDER can have only one bot. On top of that, if user does not change COMMANDER in config, the bot will not listen to his/her commands. This two conditions already make USERNAME = COMMANDER+"_bot" unique per server.
Listen, these scenarios are now quite far. The bot can run only in offline server, and what I got to know the server admins are quite hostile towards bots.
Long story short, don't spend time on this. There are much more important features to code and problems to solve. Want to collaborate on those?

**Comment by lukleh** [comment]
Fotgot to ask you, do you need to run multiple bots per server? Each bot instance consumes some considerable computer resources - memory mainly as it needs to keep the view distance grid in memory.
If after all you do need multiple bots per server, lets make it a command line switch that will turn on some randomization in the name, but I don't want to have it on by default.

**Comment by FrederickGeek8** [comment]
I wrote a little script to spawn multiple so that is why i added id_generator in the first place. (I'm making it more user-friendly then I'll most it)

Right now I am swamped with work. The only reason I worked a little on this was because I needed to test a program to see when the script crashes (dependent on the number of player_

**Comment by lukleh** [comment]
I see. Can you utilize --botname command line option? You can spawn bot with custom name.

**Comment by FrederickGeek8** [comment]
Basically my script loops and spawns the Python script on different threads, so no. I don't really know how that would work.

**Comment by FrederickGeek8** [comment]
I can't have the bot start with the players name so I did this
https://github.com/FrederickGeek8/TwistedBot/commit/10c360ff70f3f399567e5d67191c3eb348266baa

**Comment by lukleh** [comment]
closed as the proposed changes have a different purpose than corresponding code in the mainline
